scripts/prepare/addon-bundle.ts

```diff
@@ -120,13 +120,12 @@ const run = async ({ cwd, flags }: { cwd: string; flags: string[] }) => {
         ...(optimized ? dtsConfig : {}),
         entry: exportEntries,
         format: ['cjs'],
-        target: 'node18',
-        platform: 'node',
-        external: commonExternals,
+        target: 'chrome100',
```

Comment:
Would it make sense to use the same targets as our browser modules?

```suggestion
        target: browserOptions.target,
```